### ISSUE TITLE
Research the market

### ISSUE THREAD

**Issue by IvanHristov98** [storygraph/story-graph-notes#4]
Now that we have formulated the idea, let's tweak it! We still haven't researched the marked.
**Acceptance criteria**
✅ Find if there are other such products and what they already have
✅ Discover user categories
✅ Watch some story making tutorials
✅ Make an investigation template (Description below) and document it in a wiki
✅ Investigate
✅ Learn from the investigation and think up of new ideas
✅ Document the investigation and the learnings
✅ Tweak our feature list
✅ Prioritise our requirements
✅ Update wiki for requirements

The investigation template should be asking people what they are doing, what some of their common problems are, how they approach new stories and whether they are willing to use a system like Story Graph and the price they would be paying.

**Comment by IvanHristov98** [comment]
We could maybe look into:
https://www.writersstore.com/scrivener/
https://www.storyanvil.com/

**Comment by IvanHristov98** [comment]
**Investigated Scrivener**:
- lets you organise your writings by cutting your pages and misplacing them as how you find it comfortable.

**Story anvil**:
- mistook it for world anvil. It just lets you publish (and re-publish) your stories.

**World anvil**:
- let's you build worlds, by linking places
- IT'S NOT A MAP CREATION TOOL, they just let you use other images
- supports timelines, that let you track events on places and those of characters
- everything is resource driven with text descriptions and links between places
- you have to describe everything in text and it's not a simple context as our's is

**Plot factory**:
- let's you narrate your story
- exports to epub
- character and plot templates (plots are planned by answering questions)

In conclusion of the research most of the tools on the market are text driven unlike us, because our focus is to visualise the whole plot as one picture. It's highly likely that we are free of concurrence and unique to the market. ✅ 😄

**Comment by IvanHristov98** [comment]
Potential user categories:
- book & comic writers
- game plot developers
- P&P gamers
- movie directors
- actors (when they have to learn a story quickly)

**Comment by IvanHristov98** [comment]
Investigation form can be found [here](https://docs.google.com/forms/d/e/1FAIpQLSfuXGYW1mgnhutia7MBOyhE31781pOZrklWL72waa0QsUiX4w/viewform?usp=sf_link).

**Comment by IvanHristov98** [comment]
Started a BG-MAMMA [thread](https://www.bg-mamma.com/?topic=1192738) to fund the research.
